Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:16.04 as base
```

Comment:
This duplicates functionality already in the /ops directory. We generate a Dockerfile using packer, which sets all of this up using uwsgi/nginx. We also set up a database, which this appears to skip. Can you try using that setup? If there's a problem with it, I'd prefer to change the existing packer code, rather than spin off a new Dockerfile.

Reply:
@dwang159 Packer does not use the Docker cache and ends up building the entire container every time I do a `packer build -only=docker ./output/oncall.json`. It's hard to iterate on changes I'm making when the build takes 7-8 minutes to spin up - most of which is redundant minutiae, i.e. install packages, setup virtualenv, install dependencies. This is an open issue with packer, See https://github.com/cloudogu/ecosystem/issues/262 and https://github.com/hashicorp/packer/issues/1995

In a deployed scenario, the database will (should?) be deployed independently of the service. To mimic that behavior, I haven't included that as part of the service deployment.

Reply:
+1 for moving to native Dockerfile for docker build, I was being overly lazy when setting up the packer automation ;)